apps/tools-public/toolpad/resources/updateMuiPaidSupport.ts

```diff
@@ -67,6 +68,17 @@ async function updateGitHubIssueLabels(repo, issueId) {
   };
 }
 
+async function queryPurchasedSupportKey(supportKey: string) {
+  return queryStoreDatabase(async (connection) => {
+    const [rows] = await connection.execute(
+      'select count(*) as found from wp3u_x_addons where license_key = ? and expire_at > now()',
```

Comment:
This would be more correct: 

```suggestions
      `SELECT count(*) as found FROM ${STORE_TABLE_PREFIX}x_addons WHERE license_key = ? AND expire_at > now()`,
```

Reply:
that's personal preference ... keywords don't HAVE to be uppercase